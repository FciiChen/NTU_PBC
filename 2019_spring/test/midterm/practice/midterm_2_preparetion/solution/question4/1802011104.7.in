臺北開墾的歷史相當悠久，臺灣原住民分支的凱達格蘭族即世居於此。但城市規模的成形，則起始於清治時期由漢人移民建立的艋舺、大稻埕兩處聚落，以及1884年完工的臺北府城。

1875年，清治政府將大甲溪以北至基隆、原屬淡水廳管轄的地區改置臺北府，並在臺北盆地內的大加臘地區擇地興建城池作為府治，並命名為「臺北城」，「臺北」此地名始正式出現。而臺北城與鄰近的艋舺、大稻埕兩處聚落，被合稱為「三市街」，即現今的臺北舊城區。臺灣建省後，臺北取代臺南成為臺灣的行政中心，並延續迄今。

進入日治時期後，日本延續以臺北執政當局在臺灣實施市制，在1920年以「三市街」為基礎設置臺北市，並延續至今；臺北市成立的同年，轄區包含臺北、新北、基隆、宜蘭等四大地區的臺北州成立。日本執政當局對臺北市區進行大規模的公共建設，使其成為日本經略南洋的重要基地，也奠定臺北城市的發展基礎。

1945年二戰結束後，中華民國從日本手中接管臺灣，臺北市維持原制，臺北州則改制為臺北縣。1949年，中華民國政府因國共戰爭戰敗而遷移首都至臺北，臺北成為中華民國政府失去大陸領土後的統治中心；國共戰爭後，隨中華民國政府移居臺灣的大量中國大陸新住民，多數居住於臺北，一度使臺北的人口快速增加。而自1960年代以來，因為臺灣經濟的快速起飛，使得臺北的外來工作人口快速增加，而其中的大部分來自於中臺灣與南臺灣。這些外來人口除了移入臺北市，也聚居於臺北市周圍的衛星市鎮，這使得地處臺北市外圍的臺北縣人口快速攀升，並逐漸與臺北市連成一氣。

至1967年臺北市升格為直轄市之前，「臺北」的都市範圍已逐漸包括臺北市全境（當時士林、北投、內湖、南港、木柵及景美等市郊的行政區尚未併入臺北市）。但1980年代以來，臺北舊城區逐漸沒落，商業重心以忠孝東路為軸線，逐漸東移至新發展的大安區與信義區一帶，形成所謂的「東區」，取代臺北舊城區（又俗稱為「西區」）成為臺北的發展核心。而東區範圍內的信義計畫區，因為臺灣政府的大力發展，更成為現今臺北首要的中心商業區，臺北市的市政機構也在1990年代遷移至此。直至今日，臺北是臺灣人口最多的城市，也是臺灣政治、經濟與文化的中心；其城市區域的範圍，也已涵蓋了整個臺北盆地，並擴張至盆地周圍山區。中華、南洋、東洋、西洋和世界各地的文化在此兼容並蓄，不但使得臺北成為臺灣文化面向世界的櫥窗，更是華語世界的代表城市之一。

雖然臺北縣的總人口數早已達到升格直轄市的資格，但中央一直未予核准。在縣內各界長期的爭取之下，臺北縣於2010年12月25日正式改制為直轄市，並更名為「新北市」，成為一個以臺北市外圍之市鎮組成的多核心都市。

由於歷史影響，臺灣原住民語對於臺北的稱呼亦有取自臺語或日語。前者如阿美族語稱Taypak[3]，後者如達悟族語曰Taihok[4]。
BREAK