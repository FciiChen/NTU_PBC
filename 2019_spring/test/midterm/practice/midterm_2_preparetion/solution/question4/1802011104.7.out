臺_市 13
臺_的 6
9_0 4
中_民 4
北_城 4
市_圍 4
為_灣 4
的_北 4
臺_縣 4
華_國 4
